.github/workflows/python-test-package.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches:
+      - conda-1
```

Comment:
Feel free to revert this to what you had before @rosepearson , so that we can trigger it for your PR now.